Automattic/Gravatar-SDK-iOS
# Configure profile view margins with `ProfileViewConfiguration`

Closes #202 

### Description

This PR adds margin configuration option to `ProfileViewConfiguration`.

- Proposing to call the margin `padding` for discoverability, and because it's more a padding than a margin 🤔 
- Padding can be set directly to the View's property or via Configuration.
- `ProfileContentView` (`BaseProfileView`) now conforms to `UIContentView` to be able to share most of the boilerplate code.
- Subclasses will need to add just the specifics for each model they use.
- Adding a single padding default which will be shared across all subclasses, unless the subclass requires otherwise.

### Testing Steps

- CI should be green
- On the Demo app, the configuration screen has some commented code which can be used to change different configurations and see them in the list.
